histomicstk/PodoSighter_cnn_folder/Complete_CNN_Prediction.py

```diff
@@ -204,10 +237,19 @@
 f = open(args.outxml1, 'wb')
 f.write(xml_data)
 f.close()
+gc.uploadFileToItem(itemID, args.outxml1, reference=None, mimeType=None, filename=None, progressCallback=None)

```

Comment:
Why are we write `xml` annotation to the image in DSA? Do we need this?

Reply:
I wanted to get your confirmation before I remove this part. Removed in latest commit